signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -66,12 +68,22 @@ def main():
 
     print('Preprocessing pose.....')
     temp_pose_path = temp_dir / 'pose.pose'
-    preprocess_single_file(args.pose, temp_pose_path, normalization=False)
+    preprocess_single_file(args.pose, temp_pose_path, normalization=True)
 
     print('Predicting signs...')
     temp_files = []
+    start = 0
     for index, segment in tqdm(enumerate(sign_annotations)):
-        np_pose = pose_to_matrix(temp_pose_path, segment[0], segment[1]).filled(fill_value=0)
+        end = sign_annotations[index + 1][0] if index + 1 < len(sign_annotations) else None
```

Comment:
this code is very messy and not very readable, or at all documented:
what is a wide strategy? what is a tight strategy?